compiler/testData/codegen/bytecodeText/coercionToUnitOptimization/nopInlineFuns.kt

```diff
@@ -23,4 +23,4 @@ fun simpleFunVoid(f: () -> Unit): Unit {
     return f()
 }
 
-// 3 NOP
```

Comment:
Stepping through this code works beautifully without the nops, so I don't think any nops are needed. Do you see any reason there would have to be nops in the generated code for this?

Reply:
I think that's fine. @yanex could you please take a look?